ksp_plugin_adapter/localization/fr-fr.cfg

```diff
@@ -6,47 +6,47 @@ Localization {
-    #Principia_MainWindow_NewMoonAnnouncement = Avertissement: la nouvelle lune de la lunaison numéro <<1>> est arrivée ; merci de télécharger la dernière version de Principia, <<2>>.
+    #Principia_MainWindow_NewMoonAnnouncementWithKspDeprecation = Avertissement : la nouvelle lune de la lunaison numéro <<1>> est arrivée ; merci de mettre à jour KSP à la version <<3>> et de télécharger la dernière version de Principia, <<2>>. Notez que RealismOverhaul et RealSolarSystem supportent maintenant KSP <<3>>.
+    #Principia_MainWindow_NewMoonAnnouncement = Avertissement : la nouvelle lune de la lunaison numéro <<1>> est arrivée ; merci de télécharger la dernière version de Principia, <<2>>.
     #Principia_MainWindow_TargetVessel_Select = Sélectionner le vaisseau cible...
-    #Principia_MainWindow_TargetVessel_Name = Cible: <<1>>  // <<1>> vessel name
+    #Principia_MainWindow_TargetVessel_Name = Cible : <<1>>  // <<1>> vessel name
     #Principia_MainWindow_TargetVessel_Clear = Effacer
     #Principia_MainWindow_TargetVessel_SwitchTo = Échanger
     #Principia_MainWindow_PredictionSettings = Paramètres de prédiction
     #Principia_MainWindow_KspFeatures = Fonctionalités de KSP
     #Principia_MainWindow_LoggingSettings = Paramètres de logging
     #Principia_MainWindow_KspFeature_DisplayPatchedConics = Afficher les coniques juxtaposées (ne pas utiliser pour les plans de vol !)
     #Principia_MainWindow_TargetCelestial_Select = Sélectionner le corps cible...
-    #Principia_MainWindow_TargetCelestial_Name = Cible: <<A:1>>  // <<1>> celestial name
+    #Principia_MainWindow_TargetCelestial_Name = Cible : <<A:1>>  // <<1>> celestial name
     #Principia_MainWindow_TargetCelestial_Clear = Effacer
-    #Principia_MainWindow_LoggingSettings_VerboseLevel = Niveau de verbosité :
+    #Principia_MainWindow_LoggingSettings_VerboseLevel = Niveau de verbosité :
     #Principia_MainWindow_LoggingSettings_LogOption = Traces
     #Principia_MainWindow_LoggingSettings_StderrOption = stderr
     #Principia_MainWindow_LoggingSettings_FlushOption = Immédiat
     #Principia_MainWindow_LoggingSettings_RecordJournal = Enregistrer un journal (démarre au chargement)
     #Principia_MainWindow_LoggingSettings_RecordJournalResult = Le journal est <<1>>  // <<1>> ON/OFF
     #Principia_MainWindow_LoggingSettings_JournalingStatus_ON = ACTIF
     #Principia_MainWindow_LoggingSettings_JournalingStatus_OFF = INACTIF
-    #Principia_PredictionSettings_ToleranceLabel = Tolérance:
-    #Principia_PredictionSettings_ToleranceText = <<1>> m  // <<1>>: parameters.length_integration_tolerance.ToString("0.0e0").
-    #Principia_PredictionSettings_Steps = Pas:
+    #Principia_PredictionSettings_ToleranceLabel = Tolérance :
+    #Principia_PredictionSettings_ToleranceText = <<1>> m  // <<1>> : parameters.length_integration_tolerance.ToString("0.0e0").
```

Comment:
Nay, don't change the colons in the comments, the comments are in English.  Many places.